includes/admin/home/quick-links/class-sensei-home-quick-links-item.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * File containing the Sensei_Home_Quick_Links_Item class.
+ *
+ * @package sensei-lms
+ * @since $$next-version$$
+ */
+
+/**
+ * Sensei_Home_Quick_Links_Item class.
+ *
+ * @since $$next-version$$
+ */
+class Sensei_Home_Quick_Links_Item {
+
+	/**
+	 * The quick link title.
+	 *
+	 * @var string
+	 */
+	public $title;
+
+	/**
+	 * The quick link url.
+	 *
+	 * @var string
+	 */
+	public $url;
+
+	/**
+	 * Constructor for Sensei_Home_Quick_Links_Item.
+	 *
+	 * @param string $title The localised title.
+	 * @param string $url The destination url.
+	 */
+	public function __construct( string $title, string $url ) {
+		$this->title = $title;
+		$this->url   = $url;
```

Comment:
I think links like the one to install the demo course should have a special meaning anyway that will probably need to be handled in JS.

With that said, it might be a good idea to just have links like that point to "magic URLs" like "sensei://install-demo-course" or something, similar which are then handled in JS properly.

Just a random idea, as most of the time an attribute like a `dynamic` one would  be `false` anyway. :upside_down_face:

Reply:
> Should we also have some special attribute in this class, like dynamic? For the case to install a demo course.
It could also be changed when we implement the installation feature though.

I thought we would have an URL too. But to be honest I missed that option when implementing this (I missed it in the screenshots). I will give it a go. The idea from @fjorgemota is tempting but looks like "too much" maybe? Let me see 🤔 

> I also wonder about the external link icon. Do you think we should check it when consuming the API to see if it has the same URL as the site? Or should we also have an attribute for this?

Originally I thought that it should be enough to check the url in the frontend... In the end showing the icon is just about visual representation. But since there is a mapping layer and the API is just for Sensei Home... I am open to add that logic to the backend. I would say: let's try in the frontend and if it's any complicated we can move that logic to the backend.

Reply:
Finally followed @fjorgemota approach and implemented the Import Demo Course actions by using the `sensei://` protocol. We can later change this if it gets maybe too convoluted.

See 9652e1f48 and b05242593

Reply:
Just to make it very clear, the suggestion with the sensei protocol is only so we know what exactly we're expecting from that link, so the backend can "say" to the frontend: well, this one is with you. :smile: 

In other words, right now, I see the block performing something like `url === "sensei://install-demo-course"` and add the corresponding `onClick` event listener or something instead of just adding that URL to the link. It doesn't need to be something very convoluted or complex, IMHO, it's just a magic URL for us (and for the JS). :)